app/models/schema.rb

```diff
@@ -169,6 +176,13 @@
       Submit.contest_solutions(obj.id, ctx['id'])
     end)
   end
+  field :submitMetrics do
+    type !types[ContestSubmitMetricsType]
+    resolve(lambda do |obj, _args, ctx|
```

Comment:
As far as I know only owner should see metrics so you're missing permission validation here